jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/rules/EnumRule.java

```diff
@@ -216,15 +221,15 @@ protected void applyCustomizations(EnumDefinition enumDefinition, JDefinedClass
     protected EnumDefinition buildEnumDefinition(String nodeName, JsonNode node, JType backingType) {
 
       JsonNode enums = node.path("enum");
-      JsonNode javaEnumNames = node.path("javaEnumNames");
-      JsonNode javaEnums = node.path("javaEnums");
+      JsonNode javaEnumNames = pathExtensionProperty(node, "javaEnumNames");
+      JsonNode javaEnums = pathExtensionProperty(node, "javaEnums");
 
       if(!javaEnums.isMissingNode() && !javaEnumNames.isMissingNode()) {
-        System.err.println("Both javaEnums and javaEnumNames provided; the property javaEnumNames will be ignored when both javaEnums and javaEnumNames are provided.");
+        System.err.println("Both (x-)javaEnums and (x-)javaEnumNames provided; the property (x-)javaEnumNames will be ignored when both (x-)javaEnums and (x-)javaEnumNames are provided.");
```

Comment:
getExtensionPropertyNameUsed(..) will give a cleaner error message here

Reply:
Changed (and squashed away per project maintainer's guideilines)